spec/adapters/vertica_spec.rb

```diff
@@ -229,6 +235,50 @@ def VERTICA_DB.sqls
   end
 end
 
+describe "Vertica::Database#copy_into" do
+  before do
+    @db = VERTICA_DB
+    @db[:test].truncate
+  end
+
+  specify "takes data in an enumerable passed in with :data" do
+    strings = ["firstname|1"]
+    @db.copy_into(:test, data: strings)
+    expect(@db[:test].all.first.to_h).to eq({name: "firstname", value: 1})
+  end
+
+  specify "takes data from a block" do
+    strings = ["firstname|1"]
+    @db.copy_into(:test) {
+      strings.pop
+    }
+    expect(@db[:test].all.first.to_h).to eq({name: "firstname", value: 1})
+  end
+
+  specify "errors if both a block and :data are specified" do
+    expect { @db.copy_into(:test, data: ["a string"]) { "a block" } }.to raise_error(ArgumentError)
+  end
```

Comment:
It'd be worth asserting on the message here to be sure this is the ArgumentError we want.